src/components/shared/Slider/Rail.tsx

```diff
@@ -1,15 +1,533 @@
-import React, { FC } from 'react';
-import styled from 'styled-components/native';
+import React, { FC, useEffect, useMemo } from 'react';
+import { StyleProp, StyleSheet, TouchableWithoutFeedback, ViewStyle } from 'react-native';
+import styled, { DefaultTheme, ThemeProps } from 'styled-components/native';
+
+import { isNil } from 'lodash';
```

Comment:
Is lodash used within other parts of dooboo-ui-native? If so, how about changing the syntax to `import isNil from 'lodash/isNil'` to allow [tree-shaking on lodash](https://www.azavea.com/blog/2019/03/07/lessons-on-tree-shaking-lodash/)? That way we could reduce the bundle size for every user of this library.

If not, what do you think about removing lodash and implementing a helper function instead? After all, [isNil](https://github.com/lodash/lodash/blob/ddfd9b11a0126db2302cb70ec9973b66baec0975/lodash.js#L11973) is just checking for (value == null).

Reply:
Omg, I found that `@dooboo-ui/native` don't have `lodash`................
No `lodash` in the `package.json` file...LOL... I didn't know that.
I appreciate your mention!!

[babel-plugin-lodash](https://www.npmjs.com/package/babel-plugin-lodash) makes it easier to use without changing the syntax.
And `isNil` checks `null` and `undefined`. The comparison is with `==`, not `===`. :)

Thanks! I'll consider removing `lodash` for now. :(

Reply:
Yes as you can see in the [source code for lodash/isNil](https://github.com/lodash/lodash/blob/ddfd9b11a0126db2302cb70ec9973b66baec0975/lodash.js#L11973), the function itself is
```
function isNil(value) {
  return value == null;
}
```
and since undefined == null returns true, isNil checks for either null or undefined

No problem :) and thanks for teaching me about babel-plugin-lodash

Reply:
Removed importing `lodash`, and made a function `isNil` in `Rail.ts` for now.
Later, we can consider using `lodash` in `master` branch.
Thanks again!!